packages/server/src/alerts/alerts.service.ts

```diff
@@ -20,17 +26,42 @@ export class AlertsService {
           const question = await QuestionModel.findOne(payload.questionId);
 
           const queue = await QueueModel.findOne(payload.queueId);
-          if (question.closedAt || !(await queue.checkIsOpen())) {
-            console.log(`Rephrase Question alert with id ${alert.id} expired`);
+          const isQueueOpen = await queue?.checkIsOpen();
+          if (question.closedAt || !isQueueOpen) {
+            console.log(
+              `Rephrase Question alert with id ${
+                alert.id
+              } expired ${isQueueOpen} ${JSON.stringify(question)}`,
+            );
             if (!question.closedAt) {
               question.closedAt = new Date();
+              await question.save();
             }
```

Comment:
This is essentially closing a question that is stale right? 

If so, it make more senes to leave all this to the `queue-clean.service` so that we can keep track of what's happening where. 

 If you want to do it here, that's alright but you'll probably also want to change the question status to `Stale`